pkg/runs/submitter.go

```diff
@@ -439,7 +443,7 @@ func (submitter *Submitter) createReports(params utils.RunsSubmitCmdValues,
 			err = ReportJSON(submitter.fileSystem, params.ReportJsonFilename, finishedRuns, lostRuns)
 		}
 	}
-
+	//CREATE JUNIT REPORT
```

Comment:
Why in capitals ? Kindof an ugly comment. Used mixed case ?

Reply:
comment used during personal debugging, removed now